includes/enums/class-wc-youcanpay-order-action-action-enum.php

```diff
@@ -23,7 +27,11 @@ public static function get_pre_order(): string {
 		return self::$pre_order;
 	}
 
-	public static function get_all() {
+    /**
+     * @return array
+     */
+	public static function get_all(): array
```

Comment:
space 🙏

Reply:
https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/11/commits/bed2f80778af4957fea82ef435ba424548b56d3a